thirdeye/thirdeye-pinot/src/test/java/com/linkedin/thirdeye/alert/commons/TestAnomalyFetcherFactory.java

```diff
@@ -0,0 +1,21 @@
+package com.linkedin.thirdeye.alert.commons;
+
+import com.linkedin.thirdeye.alert.fetcher.AnomalyFetcher;
+import com.linkedin.thirdeye.alert.fetcher.ContinuumAnomalyFetcher;
+import com.linkedin.thirdeye.alert.fetcher.UnnotifiedAnomalyFetcher;
+import org.testng.annotations.Test;
+
+
+public class TestAnomalyFetcherFactory {
+
+  @Test
+  public void testCreateAlertFetcher() throws Exception{
+    AnomalyFetcher anomalyFetcher = AnomalyFetcherFactory.fromClassName("ContinuumAnomalyFetcher");
+    assert anomalyFetcher != null;
```

Comment:
Use TestNG's Assert.notNull instead of Java's assert, which might stop the entire program if the thrown exception is not properly handled.